config/routes.rb

```diff
@@ -13,27 +13,59 @@
 
   # Only GET works from a browser URL
   get 'example1', to: "example#example1"
+  # curl localhost:3000/example1
+
   put 'example1', to: "example#example1"
+  # curl - X PUT localhost:3000/example1
+
   post 'example1', to: "example#example1"
+  # curl - X POST localhost:3000/example1
+
   delete 'example1', to: "example#example1"
+  # curl - X DELETE localhost:3000/example1
+
+  # These examples use different HTTP verbs to route to different methods
+  put 'example', to "example#example_put"
+  # curl -X PUT localhost:3000/example
+
+  post 'example', to "example#example_post"
+  # curl -X POST localhost:3000/example
 
 
   # Examples for routes, controllers
   # and how to execute ruby in a controller
 
   # Example URL: localhost:3000/example1
   get 'example1', to: "example#example1"
+  # curl localhost:3000/example1
 
   # Example URL: localhost:3000/example2?number=100
   get 'example2', to: "example#example2"
+  # curl localhost:3000/example2
+
 
   # Example URL: localhost:3000/example3?number=100&number2=11
   get 'example3', to: "example#example3"
+  # curl localhost:3000/example3
+
+  # Example URL: localhost:3000/example3?number=100&number2=11
+  get 'but-why', to: "example#proof_you_can_name_this_whatever_123_abc"
+  # curl localhost:3000/but-why
+
 
   # More advanced ruby examples
   get 'rock_paper_scissors', controller: 'games', action: 'rock_paper_scissors'
-  get 'state_search', controller: 'search', action: 'state_filter'
+  # curl localhost:3000/rock_paper_scissors?choice=rock
```

Comment:
oh haha, I only meant for the HTTP Verbs section above.  I don't think we need curls for all of them